packages/metaconfig/versions/0.16.0.js

```diff
@@ -85,7 +85,7 @@ class ZeroSixteen {
 
     // TODO missing watch mode and other options
 
-    config.$schema = 'https://platformatic.dev/schemas/v0.17.0/' + this.kind
+    config.$schema = 'https://platformatic.dev/schemas/v0.17.1/' + this.kind
```

Comment:
It's not related with this PR at all, but I've seen an "external" test failing for this ☝🏼 
Let me know if you want me keep this change or do that in a separate PR 😉